src/librustc_lexer/src/lib.rs

```diff
@@ -1,3 +1,17 @@
+//! Simple Rust lexer.
```

Comment:
Stilistic nit, but I'd remove the word simple:

* This is full-featured real lexer for rust, it's not really simpler than the problem. I'd apply a term "simple" to something which just uses `regex`
* The word simple is repeated many times in this paragraph.